### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by aditya226369**
@anilrao316

### CODE REVIEW

File: Week-1/assignment_2/index.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                        </h1>
+                        </p>
+                        <p>Unkown Artist</p>
+                        <p>2016.Charts.100songs</p>
+                    </div>
+
+                    <div class="end-title">
+                        <div class="nav-end-button">
+                            <div class="play"><i class="fa fa-play"></i></div>
+                            <div class="nav-end-button-text">Play All</div>
```
Comment (anilrao316) [inline Week-1/assignment_2/index.html:44]: try to use BEM naming conventions for CSS u can refer http://getbem.com/naming/

**Comment by anilrao316** [comment]
Attach screenshot or recording of the working assignment

File: Week-1/assignment_2/.vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
```
Comment (anilrao316) [inline Week-1/assignment_2/.vscode/launch.json]: we can ignore this file as part of pull request

File: Week-1/assignment_2/index.css [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  background-color: #f0f0f0;
+  width: 3vw;
+  height: 90vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.body {
```
Comment (anilrao316) [inline Week-1/assignment_2/index.css:24]: we can wrap in a div and add styles to it.. if there is any other requirement it might break the ui

**Comment by anilrao316** [comment]
Browser Working Assignment : written really good appreciate!

File: Week-1/assignment_2/.vscode/launch.json (hunk 2 above)
Comment (aditya226369) [reply Week-1/assignment_2/.vscode/launch.json]: I will add the git ignore.

**Comment by aditya226369** [comment]
> Attach screenshot or recording of the working assignment

Added in README.md

**Pull request merged**